src/utils/blueprints/get-version.ts

```diff
@@ -3,19 +3,19 @@ import { decideVersion } from '@codemod-utils/blueprints';
 import type { Options } from '../../types/index.js';
 
 const latestVersions = new Map([
-  ['@babel/core', '7.22.10'],
+  ['@babel/core', '7.22.11'],
   ['@babel/plugin-proposal-decorators', '7.22.10'],
   ['@babel/plugin-transform-class-properties', '7.22.5'],
-  ['@babel/preset-typescript', '7.22.5'],
-  ['@babel/runtime', '7.22.10'],
+  ['@babel/preset-typescript', '7.22.11'],
+  ['@babel/runtime', '7.22.11'],
   ['@embroider/addon-dev', '4.1.0'],
   ['@embroider/addon-shim', '1.8.6'],
   ['@rollup/plugin-babel', '6.0.3'],
-  ['concurrently', '8.2.0'],
+  ['concurrently', '8.2.1'],
   ['ember-auto-import', '2.6.3'],
-  ['ember-cli-babel', '7.26.11'],
+  ['ember-cli-babel', '8.0.0'],
```

Comment:
Potentially breaking due to the update to `broccoli-babel-transpiler`. To be safe, I'll make a minor release.

https://github.com/babel/ember-cli-babel/releases/tag/v8.0.0